core/scripts/cleanup-LHR-for-diff.js

```diff
@@ -40,11 +35,30 @@ function cleanAndFormatLHR(lhrString) {
     entry.startTime = 0; // Not realsitic, but avoids a lot of diff churn
   });
 
-  if (extraFlag !== '--only-remove-timing') {
-    for (const auditResult of Object.values(lhr.audits)) {
+  const baseCallFrameUrl = url.pathToFileURL(LH_ROOT);
+
+  for (const auditResult of Object.values(lhr.audits)) {
+    if (!opts.skipDescription) {
       auditResult.description = '**Excluded from diff**';
     }
+    if (auditResult.errorStack) {
+      auditResult.errorStack = auditResult.errorStack.replaceAll(baseCallFrameUrl.href, '');
```

Comment:
It's still pretty of annoying to update sample LHRs for line number changes in any of the files in an error stack. Could do like `errorStack.replaceAll(/\([^)]+\)/g, '()')` but maybe we should snip it completely